.github/autobuild/windows.ps1

```diff
@@ -150,8 +150,8 @@ Function Pass-Artifact-to-Job
     {
         throw "move failed with exit code $LastExitCode"
     }
-    echo "Setting Github step output name=artifact_1::${artifact}"
-    echo "::set-output name=artifact_1::${artifact}"
+    echo "Setting Github step output name=artifact_1 to ${artifact}"
+    $Env:GITHUB_OUTPUT="artifact_1=${artifact}"
```

Comment:
This looks unexpected as it assigns the output to an environment variable instead of writing to the file referenced by that variable. Shouldn't this be some kind of redirection?

```suggestion
    echo "{artifact_1}={${artifact}}" >> "$Env:GITHUB_OUTPUT"
```

Reply:
Probably yes. Overwriting the envionment variable would be fatal ;-).